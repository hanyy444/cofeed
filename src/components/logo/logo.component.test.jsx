import { render, screen } from "@testing-library/react";
import Logo from "./Logo.component";

describe('Test Logo component', () => { 
    it('Should render without crashing', () => {
        expect.assertions(2)
        render(<Logo/>)
        const logo = screen.getByTestId('logo')
        expect(logo).toBeInTheDocument()
        expect(logo).toMatchSnapshot()
    })
 })