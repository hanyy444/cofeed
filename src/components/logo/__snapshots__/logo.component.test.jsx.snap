// Vitest Snapshot v1

exports[`Test Logo component > Should render without crashing 1`] = `
<div
  class="logo"
  data-testid="logo"
  id="logo"
>
  <h1
    class="logo__text"
  >
    Cofeed
  </h1>
</div>
`;
